packages/angular/ssr/src/routes/ng-routes.ts

```diff
@@ -31,6 +31,18 @@ import {
 } from './route-config';
 import { RouteTree, RouteTreeNodeMetadata } from './route-tree';
 
+/**
+ * The maximum number of module preload link elements that should be added for
+ * initial scripts.
+ */
+const MODULE_PRELOAD_MAX = 10;
+
+/**
+ * Regular expression used to match dynamic import statements in the form of:
+ * `import('...')` or `__vite_ssr_dynamic_import__('...')`.
+ */
+const IMPORT_REGEXP = /[import|__vite_ssr_dynamic_import__]\(['"]\.?\/(.+\.mjs)['"]\)/;
```

Comment:
nit: we can also ignore whitespaces inside of parentheses:
```suggestion
const IMPORT_REGEXP = /[import|__vite_ssr_dynamic_import__]\(\s*['"]\.?\/(.+\.mjs)['"]\s*\)/;
```

Reply:
Typescript will always remove the spaces here.